pkg/proxy/proxier.go

```diff
@@ -791,3 +793,7 @@ func (proxier *Proxier) iptablesHostPortalArgs(destIP net.IP, destPort int, prot
 	args = append(args, "-j", "DNAT", "--to-destination", net.JoinHostPort(proxyIP.String(), strconv.Itoa(proxyPort)))
 	return args
 }
+
+func isClosedError(err error) bool {
+	return strings.Contains(err.Error(), "use of closed network connection")
```

Comment:
Given your other comments, I can live with this.  Can we make it more exact than Contains?  Can we make an == check or at least a "starts with" or "ends with" ?

Also, can you include the link to the Go discussion thread as a comment here?